src/server/isomorphic.ts

```diff
@@ -95,7 +95,11 @@ export default (context: IServerContext) => {
       }))
              .then(() => {
                context.state = store.state;
```

Comment:
@devCrossNet I wonder what this line does and how it works. I wanted to implement the redirect fix according to the following instructions but couldn't figure out how to pass the router with the context.
https://github.com/vuejs/vue-router/issues/1822#issuecomment-361031675

Reply:
I think your code makes much more sense than the one from the example.

the context is the render context of the vue-ssr-renderer and if you provide a state then it automatically creates the __INITIAL_STATE__ variable on the window object.

Basically, the context is everything you can access inside the HTML template.